catalogs/catalog-jdbc-oceanbase/src/main/java/org/apache/gravitino/catalog/oceanbase/operation/OceanBaseDatabaseOperations.java

```diff
@@ -41,25 +47,56 @@ private static Set<String> createSysOceanBaseDatabaseNames() {
     return Collections.unmodifiableSet(set);
   }
 
-  @Override
-  public String generateCreateDatabaseSql(
-      String databaseName, String comment, Map<String, String> properties) {
-
-    throw new UnsupportedOperationException("Not implemented yet.");
-  }
-
   @Override
   public String generateDropDatabaseSql(String databaseName, boolean cascade) {
-    throw new UnsupportedOperationException("Not implemented yet.");
+    final String dropDatabaseSql = String.format("DROP DATABASE `%s`", databaseName);
```

Comment:
Can this method be reused?

Reply:
Fixed. I also refactored `org.apache.gravitino.catalog.jdbc.operation.JdbcDatabaseOperations#isSystemDatabase`